### ISSUE TITLE
Rendering Incompatibility with LambDynamicLights

### ISSUE THREAD

**Issue by TheDinoKingK** [Provismet/Dynamic-Fullbright#3]
### Type of Error

Incorrect Behaviour

### Mod List

LambDynamicLights (version 1.5.1)

### Description

Idk if this is something you can fix on your end but lambdynamiclights will forcefully override the entities brightness and set it back down to default values
this also doesn't change when I switch all of the ldl dynamic light emitter settings off, only when I disable ldl entirely

ldl off:
![2023-06-28_19 01 35](https://github.com/Provismet/Dynamic-Fullbright/assets/45495320/575f8eda-5293-4979-ac1d-0c6706850d29)

ldl on:
![2023-06-28_19 02 17](https://github.com/Provismet/Dynamic-Fullbright/assets/45495320/935c358a-2eb6-40ce-a270-944f05f036b0)

### Mod Version

1.1.0

### Minecraft Version

1.20 (Latest)

### Mod Loader

Quilt

### Steps to Reproduce

```bash
loading lambdynamiclights and dynamic-fullbright on quilt
you should see the issue after that in a dark place with entities
```

**Comment by Provismet** [comment]
Unfortunately there is no clean way to resolve this issue. My solution (check version 1.1.1) is simply a workaround that effectively lets the user choose.

To explain thoroughly though, both LDL and my mod are doing the following:  
- Get the light level that a mob should have.
- Modify the value.
- Return the new value.

It's a little counter-intuitive, but step 1 does not and also cannot pick up the value left by step 3 (for whichever mod happens to go first). Returning the value ejects the function entirely without letting other mods run their code too.

**In other words, only one mod can change the entity light levels at a time, the other will always be suppressed.**

My workaround was to increase the priority level of Dynamic Fullbright, so that it will always go first and set the value first. This only reverses the problem though.
So I've also added a new feature to DFb that let's you just turn off its entity light scaling if needed (thus allowing LDL to run its own entity light updates).

Hope that helps.
